docs/user/glossary.rst

```diff
@@ -43,32 +48,69 @@ so that you have a reference for how we're using them.
       Commands executed by Read the Docs when performing the build process.
       They cannot be overwritten by the user.
 
-   profile page
-      Page where you can see the projects of a certain user.
-
    project home
       Page where you can access all the features of Read the Docs,
       from having an overview to browsing the latest builds or administering your project.
 
    project page
       Another name for :term:`project home`.
 
+   reproducible
+      Being "reproducible" is an important positive quality goal of documentation.
+
+      A documentation project is said to be *reproducible* when its sources build correctly on Read the Docs over a periode of many years.
+      You can also think of being *reproducible* as being *robust* or *resillient*.
+
+      When builds are not reproducible and break due to external factors,
+      they need frequent troubleshooting and manual fixing.
+
+      The most common external factor is that new versions of software dependencies are released.
+
+      See: :doc:`/guides/reproducible-builds`.
+
+   root URL
+      Home URL of your documentation without the ``/<lang>`` and ``/<version>`` segments.
+      For projects without custom domains, the one ending in ``.readthedocs.io/``
+      (for example, ``https://docs.readthedocs.io`` as opposed to ``https://docs.readthedocs.io/en/latest``).
```

Comment:
root URL isn't changed, just moved to be alphabetical.

Reply:
This definition seems correct